interp/src/flatten/structures/environment/env.rs

```diff
@@ -192,6 +193,26 @@ pub(crate) enum CellLedger {
     Component(ComponentLedger),
 }
 
+impl Clone for CellLedger {
+    fn clone(&self) -> Self {
+        match self {
+            Self::Primitive { cell_dyn } => Self::Primitive {
+                cell_dyn: cell_dyn.clone_boxed(),
+            },
+            Self::RaceDetectionPrimitive { cell_dyn } => {
+                Self::RaceDetectionPrimitive {
+                    cell_dyn: RaceDetectionPrimitive::clone_boxed(
+                        cell_dyn.deref(),
+                    ),
+                }
```

Comment:
Not sure it matters here, but we can accomplish the same thing with
`&**cell_dyn` rather than importing `Deref`. Honestly not sure which ends up
being clearer in the long run

Reply:
Though if we rename the `clone_boxed` on `RaceDetectionPrimitive` then neither
should be necessary